From 950fdbaaf91bcc254fc0b755e35cd1c49d573b9c Mon Sep 17 00:00:00 2001
From: Roland Stigge <stigge@antcom.de>
Date: Tue, 25 Sep 2012 09:56:14 +0200
Subject: [PATCH 180/675] gpio: Document device_node's det_debounce

This patch adds documentation for set_debounce in struct device_node.

Signed-off-by: Roland Stigge <stigge@antcom.de>
Signed-off-by: Linus Walleij <linus.walleij@linaro.org>
(cherry picked from commit 1ae963143e34edaa14ab8855e0b0018c4fc77a83)

Signed-off-by: Simon Horman <horms+renesas@verge.net.au>
---
 include/asm-generic/gpio.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/asm-generic/gpio.h b/include/asm-generic/gpio.h
index 365ea09..a9432fc 100644
--- a/include/asm-generic/gpio.h
+++ b/include/asm-generic/gpio.h
@@ -60,6 +60,8 @@ struct device_node;
  * @get: returns value for signal "offset"; for output signals this
  *	returns either the value actually sensed, or zero
  * @direction_output: configures signal "offset" as output, or returns error
+ * @set_debounce: optional hook for setting debounce time for specified gpio in
+ *      interrupt triggered gpio chips
  * @set: assigns output value for signal "offset"
  * @to_irq: optional hook supporting non-static gpio_to_irq() mappings;
  *	implementation may not sleep
-- 
1.8.2

